src/bdap/vgpmessage.h

```diff
@@ -0,0 +1,197 @@
+// Copyright (c) 2019 Duality Blockchain Solutions Developers
+// Copyright (c) 2009-2019 The Bitcoin Developers
+// Copyright (c) 2009-2019 Satoshi Nakamoto
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef DYNAMIC_BDAP_VGPMESSAGE_H
+#define DYNAMIC_BDAP_VGPMESSAGE_H
+
+#include "serialize.h"
+#include "sync.h"
+#include "uint256.h"
+
+#include <map>
+#include <set>
+#include <stdint.h>
+#include <string>
+#include <vector>
+
+class CConnman;
+class CKey;
+class CKeyEd25519;
+class CNode;
+class CVGPMessage;
+
+static constexpr size_t MAX_MESSAGE_SIZE = 8192;
+static constexpr int MIN_VGP_MESSAGE_PEER_PROTO_VERSION = 71200; // TODO (BDAP): Update minimum protocol version before v2.4 release
+static constexpr size_t MAX_MESSAGE_DATA_LENGTH = 8192;
+static constexpr uint32_t MIN_CLIENT_VERSION = 2041400; // TODO (BDAP): Update minimum client version before v2.4 release
```

Comment:
2040000 ?

Reply:
These values have been reverted in this commit: https://github.com/duality-solutions/Dynamic/pull/370/commits/94146d8ce4cc2a0302bb47ef5785954607b70f30